ui/src/app/pages/server-routes/ssh-keys/ssh-keys.page.html

```diff
@@ -13,7 +13,14 @@
     <!-- always -->
     <ion-item>
       <ion-label>
-        <h2>
+        <ion-item *ngIf="!patch.data.ui['has-ssh-key']">
+          <ion-icon slot="start" color="warning" name="warning-outline" size="large"></ion-icon>
+          <h2>
+            We highly recommend adding an SSH key to your Embassy. It can be useful for debugging should anything go wrong.  We <b>do not</b> recommend using your SSH key <b>unless</b> (1) you have advanced technical knowledge of the command line or (2) are following the guidance of a Start 9 support representitive.
```

Comment:
```suggestion
            We highly recommend adding an SSH key to your Embassy. It can be useful for debugging should anything go wrong.  We <b>do not</b> recommend using your SSH key <b>unless</b> (1) you have advanced technical knowledge of the command line or (2) are following the guidance of a Start9 support representitive.
```